Data/Sys/Shaders/grade-mini.glsl

```diff
@@ -0,0 +1,594 @@
+   along with this program; if not, write to the Free Software
+   Foundation, Inc., 59 Temple Place - Suite 330, Boston, MA  02111-1307, USA.
+
+*/
+
+
+/*
+   Grade-mini (06-07-2023)
+
+   > CRT emulation shader (composite signal, phosphor, gamma, temperature...)
+   > Abridged port of RetroArch's Grade shader.
+
+
+    #####################################...STANDARDS...######################################
+    ##########################################################################################
+    ###                                                                                    ###
+    ###    PAL                                                                             ###
+    ###        Phosphor: 470BG (#3)                                                        ###
+    ###        WP: D65 (6504K)               (in practice more like 7000K-7500K range)     ###
+    ###        Saturation: -0.02                                                           ###
+    ###                                                                                    ###
+    ###    NTSC-U                                                                          ###
+    ###        Phosphor: P22/SMPTE-C (#1 #-3)(or a SMPTE-C based CRT phosphor gamut)       ###
+    ###        WP: D65 (6504K)               (in practice more like 7000K-7500K range)     ###
+    ###                                                                                    ###
+    ###    NTSC-J (Default)                                                                ###
+    ###        Phosphor: NTSC-J (#2)         (or a NTSC-J based CRT phosphor gamut)        ###
+    ###        WP: 9300K+27MPCD (8945K)      (CCT from x:0.281 y:0.311)(in practice ~8500K)###
+    ###                                                                                    ###
+    ###                                                                                    ###
+    ##########################################################################################
+    ##########################################################################################
+*/
+
+
+// Test the following Phosphor gamuts and try to reach to a conclusion
+// For GC  Japan developed games you can use -2 (Rear Projection TVs) or 2 (CRT Tube)
+// For Wii Japan developed games probably -3 (SMPTE-C), -2 (Rear Projection TVs) or 0 (sRGB/noop)
+// For Japan developed games use a temperature ~8500K (default)
+// For EU/US developed games use a temperature ~7100K
+
+
+/*
+[configuration]
+
+[OptionBool]
+GUIName = Composite Signal Type
+OptionName = g_signal_type
+DefaultValue = true
+
+[OptionRangeInteger]
+GUIName = Phosphor (-3:170M -2:RPTV-95s -1:P22-80s 1:P22-90s 2:NTSC-J 3:PAL)
+OptionName = g_crtgamut
+MinValue = -3
+MaxValue = 3
+StepAmount = 1
+DefaultValue = 2
+
+[OptionRangeInteger]
+GUIName = Diplay Color Space (0:709 1:sRGB 2:P3-D65 3:Custom (Edit L508))
```

Comment:
There's a typo here, `Diplay`.

Reply:
@AdmiralCurtiss Thanks for feedback, can you check with 19435? I want to rule out the HDR thingy implemented after it is messing with the whites. And if possible without texture mods.
Otherwise I can only add tighter clamps until next visit to my friend's house.